packages/core/test/unit/query-generator/create-database-query.test.ts

```diff
@@ -54,18 +54,17 @@ describe('QueryGenerator#createDatabaseQuery', () => {
   it('supports the charset option', () => {
     expectsql(() => queryGenerator.createDatabaseQuery('myDatabase', { charset: 'utf8mb4' }), {
       default: notSupportedError,
-      'mssql postgres': buildInvalidOptionReceivedError('createDatabaseQuery', dialectName, ['charset']),
-      snowflake: `CREATE DATABASE IF NOT EXISTS "myDatabase" DEFAULT CHARACTER SET 'utf8mb4'`,
+      'mssql postgres snowflake': buildInvalidOptionReceivedError('createDatabaseQuery', dialectName, ['charset']),
     });
   });
 
   it('supports combining all options', () => {
     const optionSupport = {
-      collate: ['postgres', 'snowflake', 'mssql'],
+      collate: ['postgres', 'mssql'],
       encoding: ['postgres'],
       ctype: ['postgres'],
       template: ['postgres'],
-      charset: ['snowflake'],
+      charset: [''],
```

Comment:
I think you can remove this line all together

Reply:
thanks, removed